src/components/pages/Users/UserPanel/UserPanel.tsx

```diff
@@ -0,0 +1,48 @@
+import * as Yup from "yup";
+
+import CleanFormikField from "components/CleanFormikField";
+import SubFormPanel from "components/SubFormPanel";
+import { PanelWidth } from "hooks/usePanel";
+
+import { FieldName, Label } from "./types";
+import type { FormFields, Props } from "./types";
+
+const schema = Yup.object().shape({
+  [FieldName.EMAIL]: Yup.string().required("Required"),
+});
+
+const UserPanel = ({ onSubmit, isSaving, ...props }: Props) => {
+  return (
+    <SubFormPanel<FormFields>
+      {...props}
+      entity="local user"
+      initialValues={{
+        [FieldName.EMAIL]: "",
+      }}
```

Comment:
Formik requires all fields to have initial values otherwise there are react errors about `A component is changing an uncontrolled input to be controlled.` which is happening when you type in the first or last name field.

If you want these blank strings to not be sent to the API then you'll need to strip the empty fields in the `onSubmit`.